main.py

```diff
@@ -25,11 +25,9 @@
     API_ID = config("API_ID", cast=int)
     API_HASH = config("API_HASH")
     SESSION = config("SESSION")
-    AUTHS = config("AUTHS")
 except BaseException as ex:
     log.info(ex)
 
-AUTH_USERS = [int(x) for x in AUTHS.split(" ")]
```

Comment:
we still want to keep AUTH_USERS

Reply:
I thought this way would be better. 

If not I can add it back